ticdc/ticdc-changefeed-config.md

```diff
@@ -118,7 +118,7 @@ protocol = "canal-json"
 terminator = ''
 # Date separator type used in the file directory. Value options are `none`, `year`, `month`, and `day`. `none` is the default value and means that the date is not separated. For more information, see <https://docs.pingcap.com/tidb/v7.0/ticdc-sink-to-cloud-storage#data-change-records>.
 date-separator = 'none'
-# Whether to use partitions as the separation string. The default value is false, which means that partitions in a table are not stored in separate directories. For more information, see <https://docs.pingcap.com/tidb/v7.0/ticdc-sink-to-cloud-storage#data-change-records)>.
+# Whether to use partitions as the separation string. The default value is false, which means that partitions in a table are not stored in separate directories. It is recommended that you set the value as `true` to avoid potential data loss in downstream partitioned tables <https://github.com/pingcap/tiflow/issues/8724>. For usage examples, see <https://docs.pingcap.com/tidb/v7.0/ticdc-sink-to-cloud-storage#data-change-records)>.
```

Comment:
```suggestion
# Whether to use partitions as the separation string. The default value is false, which means that partitions in a table are not stored in separate directories. It is recommended that you set the value as `true` to avoid potential data loss in downstream partitioned tables <https://github.com/pingcap/tiflow/issues/8724>. For usage examples, see <https://docs.pingcap.com/tidb/v7.0/ticdc-sink-to-cloud-storage#data-change-records>.
```